components/tools/OmeroJava/test/integration/BfPixelBufferTest.java

```diff
@@ -55,7 +52,6 @@ private void setUpTestFile(String fileName) throws Throwable,
     }
 
     private void tidyUp() throws Throwable {
-        FileUtils.deleteQuietly(destPath);
```

Comment:
Could use `com.google.common.io.MoreFiles.deleteRecursively` or somesuch? Not a blocker though.

Reply:
This is now removed we do not need it at all